src/Poseidon/CLI/Forge.hs

```diff
@@ -184,15 +185,23 @@ runForge (
                 Nothing -> snpSetMergeList snpSetList intersect_
                 Just _  -> SNPSetOther
     -- compile genotype data structure
-    genotypeFileData <- case outFormat of
-            "EIGENSTRAT" -> return $
+    genotypeFileData <- case (outFormat, outZip) of
```

Comment:
A bit verbose. Maybe sth like this would work?
```
gz <- if outZip then ".gz" else ""
...
GenotypeEigenstrat (outName <.> "geno" <.> gz)  Nothing
```
This would save seven lines, I think.

Reply:
Absolutely! Fixed!